src/pages/RepoPage/CoverageTab/subroute/FileExplorer/FlagMultiSelect.jsx

```diff
@@ -0,0 +1,98 @@
+import eq from 'lodash/eq'
+import isUndefined from 'lodash/isUndefined'
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+
+import { useLocationParams } from 'services/navigation'
+import { useRepoBackfilled, useRepoFlagsSelect } from 'services/repo'
+import { useFlags } from 'shared/featureFlags'
+import Icon from 'ui/Icon'
+import MultiSelect from 'ui/MultiSelect'
+
+const defaultQueryParams = {
+  // without this search here, when updating the params it fills it in for some reason :thinking:
```

Comment:
Could it be the location it's in?

Reply:
I think it's because it's parent component has a `defaultQueryParams` with just `search: ''`